# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/quota/test/xpcshell/test_initializePersistentClient.js
# Commit: aaa63956f3f0
# Full Hash: aaa63956f3f022575b29059d5909f63079723d3a
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2024-10-10 09:22:33
# Regressor Bug: 1866402
# File Overlap Count: 1
# Description:
#   Bug 1866402 - Add nsIQuotaManagerService::PersistentOriginInitialized and nsIQuotaManagerService::TemporaryOriginInitialized methods; r=dom-storage-reviewers,jari
#   
#   This additional testing infrastructure is needed for verifying origin
#   initialization status.
#   
# ==============================================================================

diff -r f799f35b1c84 -r aaa63956f3f0 dom/quota/test/xpcshell/test_initializePersistentClient.js
--- a/dom/quota/test/xpcshell/test_initializePersistentClient.js	Wed Oct 09 22:16:50 2024 +0000
+++ b/dom/quota/test/xpcshell/test_initializePersistentClient.js	Wed Oct 09 22:16:50 2024 +0000
@@ -11,7 +11,6 @@
 
 async function testSteps() {
   const clientMetadata = {
-    persistence: "default",
     principal: getPrincipal("https://foo.example.com"),
     client: "sdb",
     file: getRelativeFile("storage/permanent/https+++foo.example.com/sdb"),